Add move mesh function

mpowelson opened the pull request:
Adds a new string identifier argument to the SimDepthCamera::add. This can then be used to move the associated mesh.

mpowelson commented:
Just thought I would ping the maintainer on this. I believe this PR is needed for [Industrial Training Demo 3](https://github.com/ros-industrial/industrial_training/tree/melodic/gh_pages/_source/demo3). Currently it is pulling from a fork of mine.

marip8 commented:
@mpowelson can you rebase and convert the `Affine3d` matrix types in this PR to `Isometry3d`?

mpowelson committed:
Add move mesh function

Adds a new string identifier argument to the SimDepthCamera::add. This can then be used to move the associated mesh.
